### DISCUSSION TITLE
Windows 10 taskbar: instant crashes + battery icon

### DISCUSSION THREAD

**Discussion by radio303** [valinet/ExplorerPatcher#1988] [General]
Hello! I installed Explorer Patcher on my wife's computer to make Windows 11 22H2 look like Windows 10. Not long ago I noticed that the battery icon is hidden and there's no way to make it always seen. However, today I suddenly saw the battery icon. It turned out that Explorer Patcher disabled itself after instant crashes discussed in other threads, and because of that the battery icon reappeared. All Windows updates were applied, the latest build of Explorer Patcher was used. Is there any hope that Explorer Patcher will stop crashing Windows Explorer and the battery icon will reappear when it's used?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by radio303** [reply]
Windows 11 Home 22H2 build 22621.2215. Latest version of Explorer Patcher. I can't say what changes were made because I had to uninstall.

**Reply by pyrates999** [reply]
That is a preview update: https://support.microsoft.com/en-us/topic/august-22-2023-kb5029351-os-build-22621-2215-preview-9af25662-083a-43f5-b3a7-975fe25cc692

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

Preview updates are not supported by EP.

**Reply by radio303** [reply]
I'm sorry, it's 22621.2134 instead. I googled the latest build and got it wrong. Sorry again.

**Reply by pyrates999** [reply]
battery icon not showing, might be related to: https://github.com/valinet/ExplorerPatcher/issues/1454

crashing was reported here: https://github.com/valinet/ExplorerPatcher/issues/1828

Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu".
